tests/test_rename_associated_file.py

```diff
@@ -0,0 +1,113 @@
+# coding=utf-8
+"""Tests for medusa/post_processor.py."""
+import os
+
+from medusa import app
+from medusa.post_processor import PostProcessor as Sut
+
+import pytest
+
+
+@pytest.mark.parametrize('p', [
+    {  # p0: Subtitle with language. No subtitles dir set.
+        'new_path': 'media/shows/great show/season 1/',
+        'new_basename': 'Great Show - S01E04 - Best Episode',
+        'filepath': 'downloads/tv/great.show.s01e04.720p.hdtv.x264-group.en.srt',
+        'expected': 'media/shows/great show/season 1/Great Show - S01E04 - Best Episode.en.srt'
```

Comment:
This is already being done somewhere (didnt have the time to check where exactly). Basically if new_basename gets passed as None or an empty string. If video isn't renamed, the associated files aren't renamed either. The info can still be renamed and the language code for subs normalized tho.